source-academy/cadet-frontend
# P2Fancy: SourceAcademy GitHub Classroom with central repository

### Description



### Type of change


### How to test



1. Request to join the test GitHub classroom named 'source-academy-course-test'.
2. Request to be added as member of the GitHub Organization 'source-academy-course-test'.
3. Run Source Academy on localhost.
4. Click on Classrooms in the primary (topmost) navbar.
5. Click login and sign in with your GitHub account.
6. Select course 'source-academy-course-test'.
7. Click on Missions (there should be 3 visible mission cards with their accept buttons).
8. Click on accept for any of the missions and follow the procedure to accept the GitHub Classroom assessment.
9. Click on Missions again (the 'accept' button should change to 'open').
10. Click on Quests (there should be 1 visible quest card with accept button).

### Checklist
